torch/distributed/rpc/api.py

```diff
@@ -657,7 +657,7 @@ def _rpc_sync_torchscript(to, qualified_name, args=None, kwargs=None):
 
 
 @_require_initialized
-def _rpc_async_torchscript(to, qualified_name, args=None, kwargs=None):
+def _rpc_async_torchscript(to, user_func, args=None, kwargs=None):
```

Comment:
I think we can kill this because we can accept user_func instead of qualified name now

Reply:
Can't remove it at the moment.

`_rpc_sync_torchscript` is used in tests, `_rpc_sync_torchscript` uses `_rpc_async_torchscript`,